### DISCUSSION TITLE
Specify default desktop for applications

### DISCUSSION THREAD

**Discussion by killiandt** [lxqt/lxqt#2545] [Q&A]
Is there a way to specify which desktop an application should open on ?

For example, I'd like slack to open on desktop 1, firefox on desktop 2 ...

At least for applications in autostart.

Thanks a lot!

**Comment by stefonarch** [comment]
This depends of the window manager: kwin has window rules, openbox too (in `rc.xml` at the bottom), most wayland compositors do have that too. Not sure for xfwm4 though.

I'm using this heavily in labwc and kwin together with 3-finger swipe between 5 desktops.

**Reply by killiandt** [reply]
Thank you. I'll have a look at openbox then.
